ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -0,0 +1,42 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
```

Comment:
I think we should have something around these lines here:

```suggestion
base_images:
  ubi:
    name: ubi
    namespace: ocp
    tag: "8"
build_root:
  image_stream_tag:
    name: release
    namespace: openshift
    tag: golang-1.20
images:
- dockerfile_path: Dockerfile
  from: ubi
  inputs:
    openshift-release-golang-1.20:
      as:
      - golang:1.20
  to: network-observability-operator
operator:
  bundles:
  - as: noo-bundle
    dockerfile_path: bundle.Dockerfile
  substitutions:
  - pullspec: quay.io/netobserv/network-observability-operator:1.0.4
    with: network-observability-operator
```

Not sure about the ubi image since [we rely on docker one in Dockerfile](https://github.com/netobserv/network-observability-operator/blob/main/Dockerfile#L5).
The substitution is also hardcoded to 1.0.4 image and could be improved to [something more fancy later](https://github.com/openshift/release/blob/master/ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main.yaml#L108C16-L108C32). See [kubevirt csv](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/193d667efb3a9dc7e3a3fdb77f11e52e6b8c84ac/deploy/index-image/community-kubevirt-hyperconverged/1.11.0/manifests/kubevirt-hyperconverged-operator.v1.11.0.clusterserviceversion.yaml#L11).

Reply:
Updated the above file